src/recorder/jamrecorder.cpp

```diff
@@ -160,7 +160,7 @@ CJamSession::CJamSession(QDir recordBaseDir) :
 
     if (!fi.exists() && !QDir().mkpath(sessionDir.absolutePath()))
     {
-        throw std::runtime_error( (sessionDir.absolutePath() + " does not exist but could not be created").toStdString() );
+        throw std::runtime_error( (sessionDir.absolutePath() + " does not exist and could not be created").toStdString() );
```

Comment:
Maybe throw/catch `CGenErr`, as it's there...  Not many examples of it being used.

Reply:
Yeah...  I hadn't seen CGenErr - there aren't many, saw my first one a few weeks ago!

Reply:
Done.